Description: fix the install path 
 fix the path in Makefile to avoid installation in /usr/local
Author: Sophie Brun <sophie@freexian.com>
Origin: vendor
Last-Update: 2015-04-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Makefile
+++ b/src/Makefile
@@ -2,7 +2,7 @@ CC = gcc
 CCFLAGS = -lssl -lcrypto -Wall -Werror
 
 TARGET = pixiewps
-PREFIX = $(DESTDIR)/usr/local
+PREFIX = $(DESTDIR)/usr
 BINDIR = $(PREFIX)/bin
 
 all:
